packages/mdc-textfield/foundation.ts

```diff
@@ -481,6 +483,11 @@ export class MDCTextFieldFoundation extends MDCFoundation<MDCTextFieldAdapter> {
       value: '',
     };
   }
+
+  private hasAutofillValue_(): boolean {
+    // For testdouble
```

Comment:
I don't understand why this adapter check is needed for td... we've never had to do this before, and it looks like you're using a mock adapter properly in the tests?